fatips/0.md

```diff
@@ -246,6 +251,22 @@ described above, but with the Coinbase Address as the sole input in the
 output addresses may not intersect. Thus a Coinbase Transaction may not
 directly burn the tokens it issues.
 
+### Contract Publication Transactions
+
+Contract Publication Transactions are transactions that establish a WASM based smart contract at a given Factoid address. Establishing a contract at an address removes the address from human control and delegates control of the address to the contracts code.
+
+Contract publication transactions are identified by a zero-input zero-output transaction from the desired contract address to the burn address, while specifying the `contract` key value pair in the Transaction JSON. The `contract` field specifies the [FATIP-104](104.md) contract chain to use at the address.
```

Comment:
zero input zero output is ambiguous, does this mean an input with a 0 amount, or a tx without an input.